modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/PodcastDao.kt

```diff
@@ -417,4 +418,18 @@ abstract class PodcastDao {
         """,
     )
     abstract suspend fun getNovaLauncherSubscribedPodcasts(): List<NovaLauncherSubscribedPodcast>
+
+    @Query(
+        """
+        SELECT 
+          trending_podcast.uuid AS id, 
+          trending_podcast.title AS title 
+        FROM 
+          trending_podcasts as trending_podcast 
+          LEFT JOIN podcasts AS podcast ON podcast.uuid = trending_podcast.uuid 
+        WHERE 
+          IFNULL(podcast.subscribed, 0) IS 0
+        """,
+    )
+    abstract suspend fun getNovaLauncherTrendingPodcasts(): List<NovaLauncherTrendingPodcast>
```

Comment:
This isn't part of this PR, but I'm just wondering why we need to store the trending podcasts in the database. If we have loaded the JSON on the discover page, won't it already be cached to disk?

Reply:
How would you access this JSON in the work manager to push it to Nova?